src/dealWizard/README.md

```diff
@@ -0,0 +1,54 @@
+# Deal Wizards
```

Comment:
🤩

Reply:
Argh, but looking at this.
I'm wondering how future proof this is.
I doooo like, that we have it, but the underlying code might change fast, so this becomes outdated quickly. 🤔 

Overall, I think this still has lots of value, just because the wizard is touching multiple places, that makes 
"looking at the code" not trivial.